lib/src/mock_navigator.dart

```diff
@@ -67,13 +78,18 @@ class MockNavigator extends Mock
     with _MockNavigatorDiagnosticsMixin
     implements NavigatorState {
   /// {@macro mock_navigator}
-  MockNavigator() {
+  ///
+  /// [canPop()] is automatically stubbed to return the value of
+  /// [initialCanPop].
+  MockNavigator({bool initialCanPop = false}) {
     registerFallbackValue(_FakeRoute<dynamic>());
     registerFallbackValue(_FakeRoute<Object>());
     registerFallbackValue(_FakeRoute<void>());
     registerFallbackValue(_FakeRoute<bool>());
     registerFallbackValue(_FakeRoute<String>());
     registerFallbackValue(_FakeRoute<num>());
+
+    when(canPop).thenReturn(initialCanPop);
```

Comment:
In my honest opinion I think we should expect users to define this using the `mocktail` API without providing an additional API to go around this.

```suggestion
```

Reply:
@alestiago yeah I was a bit on the fence over whether to set it automatically or not. My reasoning for adding it in the end is that the stub is not optional; tests fail if it's not stubbed and I thought this would be preferred over all users having to update their tests to add the stub manually.

I can take it out if you disagree though, no problem 😄

Reply:
Let me gather some more opinions, before moving forward with the suggestion.

(cc: @erickzanardo , @wolfenrain , @renancaraujo , @marwfair )

Reply:
In general, I agree with making the API as easy to use as possible without adding too much obscurity or "magic" behind the scenes.

In this case though, I'm fine with it, as long as we think it won't have unintended side effects or regressions in existing user's tests.